doc/guides/west/index.rst

```diff
@@ -4,30 +4,29 @@ West (Zephyr's meta-tool)
 #########################
 
 The Zephyr project includes a swiss-army knife command line tool named
-``west``\ [#west-name]_. West is developed in its own `repository`_.  Like
-``git`` and ``docker``, the top-level ``west`` command takes some common
+``west``\ [#west-name]_. West is developed in its own `repository`_.
+
+West's built-in commands provide a multiple repository management system with
+features inspired by Google's Repo tool and Git submodules. West is also
+"pluggable": you can write your own west extension commands which add
+additional features to west. Zephyr uses this to provide conveniences for
+building applications, flashing and debugging them, and more.
+
+Like ``git`` and ``docker``, the top-level ``west`` command takes some common
 options, a sub-command to run, and then options and arguments for that
 sub-command::
 
   west [common-opts] <command> [opts] <args>
 
-West's built-in commands provide a multiple repository management
-system with features inspired by Google's Repo tool and Git
-submodules. West simplifies configuration and is also pluggable: you
-can write your own west "extension commands" which add additional
-features to west.  Zephyr uses this feature to provide conveniences
-for building applications, flashing and debugging them, and more.
+Since west v0.8, you can also run west like this::
 
-It is possible not to use west for Zephyr development if you do not
-require these features, prefer to use your own tools, or want to
-eliminate the extra layer of indirection. However, this implies extra
-effort and expert knowledge.
+  python3 -m west [common-opts] <command> [opts] <args>
 
 You can run ``west --help`` (or ``west -h`` for short) to get top-level help
```

Comment:
> Know you didn't change this, but `help` is also a built-in command, so one can do: `west help` instead of `west --help`

Yep, I know. I reviewed the PR that added this command in some detail :) https://github.com/zephyrproject-rtos/west/pull/204.

You don't seem to be asking for a change so I'm going to mark this resolved.